config/server.properties

```diff
@@ -18,7 +18,7 @@
 ############################# Server Basics #############################
```

Comment:
discard this change... if you need, change should go in Azure-Compute-Move/src/Services/Kafka/Broker/config/server.properties

Reply:
as we maintain our own server.properties in Azure-Compute-Move, see if we can keep this version in sync with apache/kafka

Reply:
I will change it back. Please focus on other code change at the moment.